about.html

```diff
@@ -0,0 +1,169 @@
+                </div>
+              </div>
+             
+             <div class="row mt-5 justify-content-center">
+              <div class="col-md-5 col-sm-10 col-xs-12">
+                <div class="lower-content">
+                  <p class="fw-bold font_size">Please contact us per email for any further questions about Educational Symposium Global Summit 2023!</p>  
+                  <p class="fw-bold font_size">educationalsymposium2023@gmail.com.</p>  
+                </div> 
+              </div>
+             </div> 
+             
+            </div>
+          </div>
+          
+        </div>
+      </section>
+      
+      <section id="main3" class="container-fluid  mt-5">
+        <div class="container p-3 mb-5">
+          <h3 id="program_heading" class="text-center">Educational Symposium Global Summit 2020 Logo</h3>
+          <hr class="hr">
+        </div>
+       
+        <div class="container">
+          <div class="row mt-5 justify-content-center">
+            <div class="col-md-5">
+              <a href="index.html">
+                <img src="images/Educational Symposium.png" alt="" class="logo-img w-100">
+              </a>
+            </div>
+          </div>
+         
+          
+        </div>
+        
+      </section>
+      <section id="main4" class="container-fluid  mt-5 border-top">
+        <div class="container p-3 mt-5 mb-5">
+          <h3 id="program_heading" class="text-center">See the past ES Global Summit</h3>
+          <hr class="hr">
+          <div class="font_size p mt-2 text-center">Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet consectetur, adipisicing elit. Excepturi aliquid aperiam debitis accusantium.</div>
+        
+        </div>
+       
+        <div class="container">
+          <div class="row mt-5 justify-content-center">
+            <div class="col-md-5">
+              <div class="past_event">
+                <h2>2020</h2>
+                <div class="h3 text-center">Educational Symposium Global Summit London.</div>
+              </div>
+            </div>
+            <div class="col-md-5">
+              <div class="past_event">
+                <h2>2022</h2>
+                <div class="h3 text-center">Educational Symposium Global Summit Paris.</div>
+              </div>
+            </div>
+          </div>
+         
+          
+        </div>
+        
+      </section>
+      <section id="partners" class="container-fluid mt-5 bg-dark">
+        <div class="container text-white p-3">
+          <h3 id="program_heading" class="text-center">Partners</h3>
+          <hr class="hr">
+        </div>
+        <div class="container">
+          <div class="row">
+            <img src="images/airbnb" alt="" class="social_links w-100 col">
+            <img src="images/mozilla" alt="" class="social_links w-100 col">
+            <img src="images/naver.png" alt="" class="social_links w-100 col">
+          </div>
+        </div>
+      </section>
```

Comment:
- This section is only present in the mobile version of the About page and not in the desktop version, please fix this 🙏
- Also, please make sure that in the mobile version, the section is aligned as it is in the design template. Please fix this 👍 👍